### ISSUE TITLE
File must end with a newline with empty files

### ISSUE THREAD

**Issue by pgreze** [pinterest/ktlint#723]
## Expected Behavior

This is expecting for a build.gradle.kts file with no content.
Expected behavior should be no error.

## Observed Behavior

```
.../build.gradle.kts:1:1: File must end with a newline (\n)
```

## Your Environment

* Version of ktlint used: id("org.jlleitschuh.gradle.ktlint") version "9.2.1"
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Gradle plugin
* Version of Gradle used (if applicable): 6.0.1
* Operating System and version: OSX
* Link to your project (if it's a public repository): https://github.com/pgreze/kowners
